### ISSUE TITLE
Usability QoL items

### ISSUE THREAD

**Issue by rshih32** [truera/trulens#16]
- [ ] move quick start to top and add before hard to see badges "Quickstart Pytorch, and Tensorflow/Keras notebooks are below:"
- [ ] In the notebooks, make clear attribution shape is input shape and that mask visualizer is just a helper for image data and not mandatory
